src/main/java/org/kohsuke/github/GHEvent.java

```diff
@@ -85,4 +87,50 @@ String symbol() {
             return "*";
         return name().toLowerCase(Locale.ENGLISH);
     }
+
+    /**
+     * Representation of GitHub Event Type
+     *
+     * @see <a href="https://docs.github.com/en/developers/webhooks-and-events/github-event-types">GitHub event
+     *      types</a>
+     */
+    public enum GitHubEventType {
```

Comment:
I don't think this needs to be public.  It is only used like an private`HashMap<String, GHEvent>`.  
```suggestion
    enum GitHubEventType {
```